### ISSUE TITLE
Please add WAITS to the selection of PDP-10 operating systems

### ISSUE THREAD

**Issue by larsbrinkhoff** [obsolescence/pidp10#14]
There are three sets of disk images here:  
https://www.sky-visions.com/dec/waits/

I don't know which is what.  I would recommend the wabbn.ini configuration, since it's closer to how WAITS developed later in life.  We should check that III and DD displays are available.

CC @rcornwell, @bgbaumgart

**Comment by bgbaumgart** [comment]
Lars
-
I have been wasting my  time (and some money) on Vision Pro modeling of the SAIL DCPower Building and grounds for 1972.
I will try to get the many WAITS disk images out to GITHUB this month “as-is-where-is" as a starting point for the WAITS (aka WTS) Mausoleum.
-
Bruce

Snap shot thru my Vision Pro — model of A.I. lab building with glowing roof rafters.
￼



Recent 2024 snap shot of me time shifted into the. PDP-1 room at Harvard 1968 where I did head motion tracking with professor Ivan Sutherland. 

￼

**Comment by rcornwell** [comment]
There are waits images on my web site. However there is very little docs on them. @bgbaumgart can you provide a list of logins and passwords for this disk set?

**Comment by larsbrinkhoff** [comment]
CC @drfuchs, who wants to run WAITS.

**Comment by larsbrinkhoff** [comment]
Regarding logins, those are kind of the same as the directories, right?  As for passwords, maybe they are not needed when logging in "locally" (i.e. not from network, or off-site teletype, etc).

**Comment by drfuchs** [comment]
Yup.  On Waits, login names were 3 characters, typically your initials (so I was DRF and Knuth was DEK for instance). And there was exactly one level of directory structure after that:  You got to name each of your personal top-level directories with 3 prefix characters; for instance, including syntactic sugar, all the TeX code is in [TEX,DEK].  This means a directory can be specified in 6 upper-case characters, which conveniently fit into a single 36-bit word.  By convention, your main directory was named "1" (so, e.g.,  [1,DRF]).  Filenames are then limited to 6 characters plus a 3 character extension (leaving another 18 bits for protection bits and other flags, I suppose).  I have no clue how the sysadmins added user accounts and passwords, never mind how they logged in and to what account, so that might present a challenge. You can see the semi-final state of Sail in places like https://www.saildart.org/%5BTEX,DEK%5D/
Apologies if this is sufficiently like ITS to be obvious.  Oh, if there was a time where people didn't use passwords on Sail, it must have been before I arrived in 1978.  (You could peek at everyone else's screen, though, courtesy of a cross-bar switch on the video output channels of the DD controller; of course, you could issue a command to "hide" your screen.)
-drf
    On Tuesday, June 11, 2024 at 08:49:00 AM EDT, Lars Brinkhoff ***@***.***> wrote:  
 
 


Regarding logins, those are kind of the same as the directories, right? As for passwords, maybe they are not needed when logging in "locally" (i.e. not from network, or off-site teletype, etc).

**Comment by rcornwell** [comment]
Adding waits to PiDP10 is not hard. The disks are available at my web site. However we don't know many of the login passwords. So it is hard to do anything. Also the system is kind of a cut and pasted disk image, so it might be hard to reconfigure it for custom changes.
